app/templates/main/volunteerProfile.html

```diff
@@ -53,14 +53,11 @@ <h6 align="center">{{volunteer.phoneNumber}}</h6>
            </tr>
        {% endfor %}
      {% else %}
-       <h2><p class="text-dark" align="center">Choose interests below to be notified about upcoming events.</p><h2>
+       <h4><p class="text-dark" align="center"> No upcoming events. </p></h4>
+       <h4><p class="text-dark" align="center">Choose interests below to be notified about upcoming events.</p><h4>
```

Comment:
Is there a reason we are using a paragraph tag inside a header tag? It seems a little backwards. You should be able to get the styles you want with just a header or just a paragraph

Reply:
The size seems good right now. I'm not sold on the center alignment, though.